packages/ui-tests/cypress/e2e/designer/specialCamelRoutes/routeConfiguration.cy.ts

```diff
@@ -3,26 +3,40 @@ describe('Test for root route configuration container', () => {
     cy.openHomePage();
   });
 
+  it('Update routeConfiguration description', () => {
+    cy.selectCamelRouteType('Configuration', 'routeConfiguration');
+
+    // Open routeConfiguration configuration tab, using the first routeConfiguration node
+    cy.get(`g[data-nodelabel^="routeConfiguration-"]`).eq(0).click({ force: true });
+    cy.selectFormTab('All');
+    cy.interactWithConfigInputObject('description', 'routeConfigurationDescription');
+    cy.closeStepConfigurationTab();
+
+    cy.openStepConfigurationTab('routeConfigurationDescription');
+  });
+
   it('Root route configuration', () => {
     cy.selectCamelRouteType('Configuration', 'routeConfiguration');
 
-    cy.openStepConfigurationTab('routeConfiguration');
+    // Open routeConfiguration configuration tab, using the first routeConfiguration node
+    cy.get(`g[data-nodelabel^="routeConfiguration-"]`).eq(0).click({ force: true });
```

Comment:
When adding a new `RouteConfiguration`, the node label uses the `id`, so the label is `routeConfiguration-1234`, therefore our current logic to open it doesn't work because there's no exact match, that's why I'm opening it by hand.